packages/auth/gulpfile.js

```diff
@@ -75,11 +105,13 @@ gulp.task('clean', done => del(['dist/*', 'dist'], done));
 gulp.task('serve', () => {
   const app = express();
 
-  app.use('/node_modules', express.static(path.resolve(__dirname, '../../node_modules')));
+  app.use(
+    '/node_modules',
```

Comment:
indent 2 additional spaces. Style guide recommendations is to indent 4 spaces for function arguments that spill over to next line.